keyboard/src/modules/Keyboard.rb

```diff
@@ -281,6 +282,21 @@ def Import(settings, syntax = :keyboard)
       when :language
         keyboard = GetKeyboardForLanguage(settings["language"], keyboard)
       end
+
+      # Checking if the keymap exists. Either it is the real keymap name
+      # or an alias.
+      if !Keyboards.code(keyboard)
+        # Checking if it a real keymap name
+        checked_keyboard = keyboard
+        keyboard = Keyboards.alias(checked_keyboard)
+        if !keyboard
+          # TRANSLATORS: the "%s" is the kaymap name
+          Report.Warning(_("Cannot find keymap: %s. Taking default one.") % checked_keyboard)
```

Comment:
The question could be, what is the default one? We know it, of course. But, what about the user?

Reply:
To be honest at that moment even we do not know it because the proposal will be called afterward. :-)